packages/notifi-react-hooks/lib/hooks/useNotifiClient.ts

```diff
@@ -126,6 +126,8 @@ const signMessage = async ({
   timestamp: number;
 }>): Promise<string> => {
   switch (params.walletBlockchain) {
+    case 'XION':
+      throw new Error('XION not supported with react-hooks, please migrate to Notifi Client.');
```

Comment:
the term `Notifi Client` might be confusing to users. 
I would use `XION not supported with react-hooks, please enable frontend-client (isUsingFrontendClient={true}`.